cache/remote.go

```diff
@@ -23,14 +24,14 @@ type Unlazier interface {
 	Unlazy(ctx context.Context) error
 }
 
-func (sr *immutableRef) GetRemote(ctx context.Context, createIfNeeded bool, compressionType compression.Type) (*solver.Remote, error) {
+func (sr *immutableRef) GetRemote(ctx context.Context, createIfNeeded bool, compressionType compression.Type, s session.Group) (*solver.Remote, error) {
```

Comment:
Why isn't session ID passed to `lazyRefProvider` in this function?

https://github.com/moby/buildkit/blob/b83d12c400d33f54c724d7d56d6a25fa0d4d9fa8/cache/remote.go#L105-L109

Reply:
good catch. updated